all-actors/src/main/java/org/sunbird/serviceimpl/CertsServiceImpl.java

```diff
@@ -233,6 +244,44 @@ public Response downloadV2(Request request) throws BaseException {
         return response;
     }
 
+    private String getJsonSignedUrl(String jsonUrl) throws BaseException {
```

Comment:
Do we really need to write this code again?
I am not sure if this is very different from 'download' function.

Reply:
done, using download' function.